bokeh_root_cmd/main.py

```diff
@@ -1,61 +1,156 @@
-import os
+"""Command line wrapper to serve one or more named Bokeh scripts or folders."""
 import logging
+import os
+import pathlib
+from typing import Any, Dict, Tuple
 
-from bokeh.command.util import build_single_handler_application
-from bokeh.server.server import Server
+import bokeh.server.views
 import click
+from bokeh.application.application import Application
+from bokeh.command.util import build_single_handler_application
+from bokeh.server.server import Server as _BkServer
+from bokeh.server.views.root_handler import RootHandler
+from panel.io.server import INDEX_HTML as _PANEL_INDEX_HTML
+from panel.io.server import Server as _PnServer
+import logging
 
 from .readycheck import create_ready_app
 
-class BokehException(Exception):
-    pass
+FORMAT = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
 
-def make_app(command, debug=False):
+logging.basicConfig(format=FORMAT)
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
 
-    # Command can be absolute, or could be relative to cwd
-    app_py_path = os.path.join(os.getcwd(), command)
+logger = logging.getLogger('bokeh_root_cmd')
 
-    print("Fetching Bokeh script or folder {}".format(app_py_path))
+_BOKEH_INDEX_HTML = str(pathlib.Path(bokeh.server.views.__file__).parent / "app_index.html")
```

Comment:
Please could these HTML extractions happen inside the Bokeh/PanelServer classes?

So maybe there is a get_index_html() static method that performs this operation (and can 'cache' it in a static property too).

Of course this would save performing the operation when it's not needed. Partly, I suggest this for speed, but more importantly in case something goes wrong - especially if a bokeh user doesn't quite have the right panel installed (or has none at all), so the import breaks (when it wasn't actually needed anyway!)

I also have a related note about imports and requirements which I'll add below.